app/pages/profile/comment-link.jsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+import apiClient from 'panoptes-client/lib/api-client';
+import { Markdown } from 'markdownz';
+import { Link } from 'react-router';
+
+class CommentLink extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      projectTitle: ''
+    };
+  }
+
+  componentDidMount() {
+    this.getProject(this.props.comment);
+  }
+
+  componentDidUpdate(prevProps) {
+    if (prevProps.comment.section !== this.props.comment.section) {
+      this.getProject(this.props.comment);
+    }
+  }
+
+  getProject(comment) {
```

Comment:
If project title was serialised with the comment, this API call wouldn't be necessary and this could be a purely functional component.